### PR TITLE
lint: use flake8-use-fstring plugin

### PR DESCRIPTION
**Pull request by maratori**
Let's use [flake8-use-fstring](https://github.com/MichaelKim0407/flake8-use-fstring)

### CODE REVIEW

File: lona/response_parser.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def render_response_dict(self, raw_response_dict, view_name):
             # check keys
             if len(set(raw_response_dict) & key_words) != 1:
                 raise RuntimeError(
-                    'response dicts have to contain exactly one keyword of: {}'.format(  # NOQA: LN001
+                    'response dicts have to contain exactly one keyword of: {}'.format(  # NOQA: LN001,FS002
```
Comment (fscherf) [inline lona/response_parser.py:53]: Why do we leave this line like this?

File: lona/static_files.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def __init__(self, name, path, url=None, sort_order=None, link=True,
         self.static_url_prefix = ''
 
     def __repr__(self):
-        return '<{}({}, sort_order={}, enabled={}, {}{} -> {})>'.format(
+        return '<{}({}, sort_order={}, enabled={}, {}{} -> {})>'.format(  # NOQA: FS002
```
Comment (fscherf) [inline lona/static_files.py:27]: Why do we leave this line like this?

File: lona/response_parser.py (hunk 1 above)
Comment (maratori) [reply lona/response_parser.py:53]: While migrating to f-strings in #58 I decided that in some cases `.format` is more readable.
All such cases now have `# NOQA`.
If you think it's better to rewrite them using f-strings, I can update PR.

File: flake8.ini [hunk 3]
```diff
@@ -14,8 +14,12 @@ per-file-ignores =
     # imported but unused
     __init__.py: F401
 
-# require code(s) to be included in # noqa comments
+# flake8-noqa: require code(s) to be included in # noqa comments
```
Comment (fscherf) [inline flake8.ini:17]: This should be it's own commit, or the change should be at least mentioned in the commit message

**Review by fscherf — CHANGES_REQUESTED** [review]

File: flake8.ini (hunk 3 above)
Comment (maratori) [reply flake8.ini:17]: Fixed

**Review by fscherf — APPROVED** [review]
Great! Thanks! :)
